README.md

```diff
@@ -1,4 +1,4 @@
-# ChangesReloaded ![Build Status](https://github.com/hpi-swa-teaching/ChangesBrowser/workflows/CI/badge.svg?branch=dev)[![Coverage Status](https://coveralls.io/repos/github/hpi-swa-teaching/ChangesBrowser/badge.svg?branch=dev)](https://coveralls.io/github/hpi-swa-teaching/ChangesBrowser?branch=dev)
+# ChangesReloaded [![Build Status](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser.svg?branch=team09)](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser)
```

Comment:
```suggestion
# ChangesReloaded [![Build Status](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser.svg?branch=team06)](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser)
```
Nicht Team 06?

Reply:
noooo this is wrong all along. make this come back
```suggestion
# ChangesReloaded ![Build Status](https://github.com/hpi-swa-teaching/ChangesBrowser/workflows/CI/badge.svg?branch=dev)[![Coverage Status](https://coveralls.io/repos/github/hpi-swa-teaching/ChangesBrowser/badge.svg?branch=dev)](https://coveralls.io/github/hpi-swa-teaching/ChangesBrowser?branch=dev)
```